65black/so-lite-react
# #162832571 Add Functionality to Post an Answer

#### What does this PR do?
This PR adds the ability for a logged in user to post an answer to a question by filling a form.

#### Description of Task to be completed?
- add tests for new functionality

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Log in to your account #3 
- Navigate to `localhost:7000/questions` and click on any question
- Scroll to the bottom and fill in the form to add an answer
- If successful, the page will reload and your answer will appear amongst the answers list

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832571

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-28 at 10 56 16 am" src="https://user-images.githubusercontent.com/36507972/50511820-6a357480-0a8f-11e9-9ce4-cc7d222e9e60.png">
<img width="1440" alt="screenshot 2018-12-28 at 10 55 52 am" src="https://user-images.githubusercontent.com/36507972/50511821-6ace0b00-0a8f-11e9-974f-cdec40f3eedd.png">

#### Questions:
N/A
